components/concept/fetch/src/main/java/mozilla/components/concept/fetch/Response.kt

```diff
@@ -64,7 +82,7 @@ data class Response(
          * Executes the given [block] function with the buffered reader as parameter and then closes it down correctly
          * whether an exception is thrown or not.
          */
-        fun <R> useBufferedReader(charset: Charset = Charsets.UTF_8, block: (BufferedReader) -> R): R = use {
+        fun <R> useBufferedReader(block: (BufferedReader) -> R): R = use {
```

Comment:
What if the server does not return a `Content-Type` header but I want to read the response with a specific charset (because I expect something specific). Does it make sense to still allow to override the charset here (Instead of UTF_8 by default make it nullable)?

Reply:
OK, I will add that.